.azure-pipelines.yml

```diff
@@ -213,7 +213,7 @@ jobs:
     displayName: 'Install'
   - bash: |
       set -eux
-      py.test $(pytest.base_args) tests -n auto --runrust
+      python tests/run_parallel.py -n auto $(pytest.base_args) tests --runrust
```

Comment:
Why is `-n auto` still here if `run_parallel.py` already runs the tests in parallel?

Reply:
Nevermind, I get it now.